src/utils.ts

```diff
@@ -31,40 +31,66 @@ export const libraryToString = ({
 // the future.
 () => semantics;
 
+const filterWord = (word: string): string => {
+  const variantIndex = word.indexOf("/");
+  const tipsIndex = word.indexOf(":");
```

Comment:
Do you think creating constants for the separator characters would be a good idea? They seem mostly set, but in case they were to change it would be nice to keep them all in sync.